doc/usage.rst

```diff
@@ -29,6 +29,37 @@ the OpenML Python connector, followed up by a simple example.
 
 * `Introduction <examples/introduction_tutorial.html>`_
 
+~~~~~~~~~~~~~
+Configuration
+~~~~~~~~~~~~~
+
+The configuration file resides in a directory ``.openml`` in the home
+directory of the user and is called config. It consists of ``key = value`` pairs
+which are separated by newlines. The following keys are defined:
+
+* apikey:
+    * required to access the server
+
+* server:
+    * default: ``http://www.openml.org``
+
+* cachedir:
+    * if not given, will default to ``file.path(tempdir(), "cache")``.
+
+* avoid_duplicate_runs:
+    * if not given, will default to ``True``.
```

Comment:
Explain what this does, i.e. when `run_flow_on_task` or similar methods are called, when set to `True` a lookup is performed to see if there already exists such a run on the server, and if so download those results instead.